@remirror/showcase/src/social.tsx

```diff
@@ -37,6 +37,8 @@ const userData: UserData[] = fakeUsers.results.map(
   }),
 );
 
+export { fakeUsers, fakeTags, userData };
```

Comment:
This is the only change that should be necessary to the `@remirror` folder; I'd like to find a way to remove the need for the above changes.